relays/lib-substrate-relay/src/finality/mod.rs

```diff
@@ -51,14 +50,7 @@ pub(crate) const RECENT_FINALITY_PROOFS_LIMIT: usize = 4096;
 
 /// Substrate -> Substrate finality proofs synchronization pipeline.
```

Comment:
@serban300 
I see you splitted `SubstrateFinalitySyncPipeline` to `SubstrateFinalityPipeline` and `SubstrateFinalitySyncPipeline`
but both have the same docs now:
```
/// Substrate -> Substrate finality proofs synchronization pipeline.
```

there is a distinction of `Sync` word, so it should probably reflect the docs somehow,